project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.9
```

Comment:
yes, I got some weird stuff happening with 15. Apparently they started adding support for Java 9 and it messed up with the `sbt "export test:dependency-classpath" --error` command.
Specifically this is where it was blowing up I believe `https://github.com/sbt/sbt-launcher-package/blob/master/java9-rt-export/src/main/java/io/github/retronym/java9rtexport/Export.java`